qpc/utils.py

```diff
@@ -465,7 +463,7 @@ def read_in_file(filename):
                 result = in_file.read().splitlines()
         except EnvironmentError as err:
             err_msg = t(messages.READ_FILE_ERROR % (input_path, err))
-            log.error(err_msg)
+            logger.error(err_msg)
```

Comment:
It looks like `t` is just a renamed `_` (the standard translation function), and it looks like `err_msg` is only used once here. So, to follow the new pattern we've established throughout this PR, I think these lines should also change to something like:

```python
logger.error(t(messages.READ_FILE_ERROR), input_path, err)
```

And if you have time and aren't exhausted from these changes yet, you could also update the definition of `READ_FILE_ERROR` to include placeholder tokens like this:

```python
READ_FILE_ERROR = "Error reading from %(path)s: %(error)s."
```

…which could result in an even better:

```python
logger.error(t(messages.READ_FILE_ERROR), {"path": input_path, "error": err})
```

Reply:
I tried to get rid of this t in the past for consistency in code but it broke a lot of tests. I will address this in a later opportunity.